ariadne/cardano/src/Ariadne/MainTemplate.hs

```diff
@@ -116,7 +116,7 @@ initializeEverything MainSettings {..}
   let
     mkWalletFace :: (Doc -> IO ()) -> WalletFace
     walletInitAction :: IO ()
-    (mkWalletFace, walletInitAction) =
+    (mkWalletFace, walletInitAction, walletAction) =
```

Comment:
This conflicts with https://github.com/serokell/ariadne/pull/364. One of you will have to resolve some merge conflicts.

Reply:
#364 has been merged.